Enhancement: allow multiple filename attachment exclusion patterns for a mail rule (#5524)
paperless-ngx/paperless-ngx: docs/usage.md

Paperless will check all emails only once and completely ignore messages
that do not match your filters. It will also only perform the rule action

> on e-mails that it has consumed documents from. The filename attachment
> exclusion pattern could be splitted by commto to allow different exclusion
> patterns.

Reviewer: ```suggestion
on e-mails that it has consumed documents from. The filename attachment
exclusion pattern can include multiple patterns separated by a comma.
```
